core/src/main/kotlin/ch/ergon/dope/resolvable/clause/model/JoinClause.kt

```diff
@@ -97,7 +113,15 @@ sealed class SelectJoinClause : ISelectJoinClause {
             is AliasedBucket -> joinable.asBucketDefinition().toDopeQuery(manager)
             else -> joinable.toDopeQuery(manager)
         }
-        val joinQueryString = "${parentDopeQuery.queryString} ${joinType.type} ${joinableDopeQuery.queryString}"
+        val joinHints = if (hashOrNestedLoopHint != null || keysOrIndexHint != null) {
+            "USE" +
+                hashOrNestedLoopHint?.let { " ${hashOrNestedLoopHint.queryString}" }.orEmpty() +
+                keysOrIndexHint?.let { " ${keysOrIndexHint.toDopeQuery(manager).queryString}" }.orEmpty()
```

Comment:
We don't need the params defined inside _keysOrIndexHint_?

Reply:
I added them, thanks for catching that!